app/src/main/res/layout/clock_fragment.xml

```diff
@@ -17,11 +18,29 @@
     <TextClock
             android:id="@+id/text_date_id"
             android:layout_width="match_parent"
-            android:layout_height="match_parent"
+            android:layout_height="wrap_content"
             android:format12Hour="@string/clock_format_date_12_hour"
             android:format24Hour="@string/clock_format_date_24_hour"
             android:textSize="@dimen/clock_text_date_size"
             android:textColor="@color/secondaryViewColor"
             android:gravity="center_horizontal"/>
 
+    <TextClock
+            android:id="@+id/text_location_id"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:textSize="@dimen/clock_text_date_size"
+            android:textColor="@color/secondaryViewColor"
+            android:gravity="center_horizontal"/>
+
+    <ImageButton
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:layout_marginTop="300dp"
```

Comment:
Don't use such things! No any marginTop="300dp" in linearLauout  ever!
If you need to place button in the bottom of screen - use appropriate layout, like Relative or Frame